Sources/MapboxCoreNavigation/LegacyRouteController.swift

```diff
@@ -5,8 +5,8 @@ import Foundation
 import CoreLocation
 import MapboxDirections
 import Polyline
-import MapboxMobileEvents
 import Turf
+import UIKit
```

Comment:
Why is it added here?

Reply:
`UIDevice` reference. `UIKit` was imported transitively through MME.